modules/linear_boltzmann_solvers/lbs_solver/lbs_solver.h

```diff
@@ -63,6 +63,8 @@ class LBSSolver : public opensn::Solver
 
   void SetBoundaryOptions(const InputParameters& params);
 
+  double ComputePointwisePhiChange(LBSGroupset& groupset);
```

Comment:
This should be moved to more like routines instead of being embedded within the getters/setters. Maybe near the other post-iteration routines.